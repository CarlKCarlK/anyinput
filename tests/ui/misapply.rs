use anyinput::anyinput;

#[anyinput]
struct Test {
    a: AnyString,
    b: AnyString,
}

fn main() {}
