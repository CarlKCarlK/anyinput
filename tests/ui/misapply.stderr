error: expected `fn`
 --> tests/ui/misapply.rs:4:1
  |
4 | struct Test {
  | ^^^^^^
